setup-production.adoc

```diff
@@ -347,10 +372,37 @@ mv settings/config.py.prod.example settings/config.py
 
 Edit it with your own configuration:
 
-- connection to PostgreSQL
-- connection to RabbitMQ for `taiga-events` and `taiga-async`
-- credentials for email
-- Enable/disable anonymous telemetry
+- Taiga secret key: **it's important** to change it. It must have the same value as the secret key in `taiga-events` and `taiga-protected`
+- Taiga urls: configure where Taiga would be served using `TAIGA_URL`, `SITES` and `FORCE_SCRIPT_NAME` (see examples below)
+- Connection to PostgreSQL; check `DATABASES` section
+- Connection to RabbitMQ for `taiga-events`; check "EVENTS" section
```

Comment:
There is an "Events" section in both the "config.py" you're updating, and the documentation you're reading. Perhaps it could be fine to clarify it refers to the configuration file itself.